sources/packages/backend/apps/db-migrations/src/sql/Restrictions/Update-and-add-restrictions.sql

```diff
@@ -0,0 +1,27 @@
+-- Update AV, 5, 7 and 9 restriction code.
```

Comment:
IMO the file name seems to be very generic and could be more contextual. (e.g. add-bclm-restriction-and-update-actions).